docs/docs/aleph_zero/sdk.md

```diff
@@ -10,31 +10,55 @@ SDK (Software Development Kit) can be used to interact with our DEX programatica
 
-Invariant SDK consists of 3 separate contracts: our DEX contract (Invariant), token contract (PSP22), contract that can be used to wrap native currency (Wrapped AZERO), various values and helper functions. You can deploy your own or use an existing contract.
+1. DEX Contract (Invariant): This is the core contract handling DEX functionality within the Invariant ecosystem.
 
-### Txs and Queries
+2. Token Contract (PSP22): This contract is responsible for managing and implementing the PSP22 token standard within the Invariant framework. Allows to deploy or load existing contracts.
 
-After you load or deploy a contract you can start interacting with it. In order to make a contract call you have to use methods of class that represents specific contract. First parameter will always be an account you want to use and rest of parameters will be an entrypoint parameters.
+3. Wrapped Native Currency Contract (Wrapped AZERO): This contract allows users to wrap the native currency, providing compatibility and interoperability within the Invariant ecosystem.
 
-### Event listeners
+### Transactions and Queries
 
-Invariant class has additional methods that allows you to attach event listeners so you can run your own code based on contract activity.
+When working with contracts, developers can seamlessly initiate interactions by calling methods from the corresponding contract class. The first parameter designates the account, and subsequent parameters act as entrypoint parameters, facilitating smooth contract interactions.
+
+1. **Transactions**: These involve invoking methods that result in changes to the blockchain state. Transactions typically alter the data stored on the blockchain and may include operations like transferring assets, updating records, or executing specific actions.
+
+2. **Queries**: Unlike transactions, queries are read-only interactions with the contract. They allow developers to retrieve information from the blockchain without modifying its state. Queries are useful for obtaining current contract state, fetching data, or verifying certain conditions without making any changes to the blockchain.
+
+### Listening to Events
+
+The Invariant class offers additional methods empowering developers to attach event listeners. This feature enables the execution of custom code based on contract activity, enhancing flexibility and customization in response to specific events.
 
 ### Values and Helper functions
 
-SDK also contains all essential values and helper functions that might be needed in your app like maximum tick, maximum price, calculating price impact and many others that will simplify your calculations.
+The SDK includes fundamental values and utility functions for application development. These encompass parameters such as maximum tick, maximum price, and calculations for price impact. This suite of utility functions streamlines computational aspects, enhancing the efficiency of application logic.
```

Comment:
The entire documentation includes marketing-like sentences that contribute minimal technical value to understanding docs. For instance, in the following statement: "This suite of utility functions streamlines computational aspects, enhancing the efficiency of application logic". Try to be concise and direct.

Please review the entire docs and minimize the occurrence of such examples.